pyproject.toml

```diff
@@ -34,6 +34,12 @@ dependencies = [
     'lxml',
 ]
 
+[project.optional-dependencies]
+dev = [
+    "black",
+    "isort"
+]
+
```

Comment:
Without using additional dependency manager tool, that's it. You define [optional deps](https://peps.python.org/pep-0631/#optional-dependencies).
With an external tool, well, poetry.

By the way, I modified the Makefile as you directly called kalamine which was installed in my python. As I can't understand if you use virtualenvs for tests, I simply used `python -m kalamine.cli` not to cath the installed kalamine executable. That could be the subject of another PR, but again, I need to uderstand what kind of system-wide tests you could do (By the way, the tests are broken :s).

Reply:
The tests work… if you call them with a `make test`, because they expect the layouts to be built. I know it’s not how it’s supposed to work — if you can think of a better way and include that in the CI, your help would be very welcome.

Reply:
Ok. I understand. Tests import again my system-wide kalamine.